tests/kickstart_tests/timezoneLOCAL.ks

```diff
@@ -0,0 +1,39 @@
+#version=DEVEL
+url --url="http://dl.fedoraproject.org/pub/fedora/linux/development/$releasever/$basearch/os/"
+install
+network --bootproto=dhcp
+
+bootloader --timeout=1
+zerombr
+clearpart
```

Comment:
We should be consistent about using zerombr/clearpart in our kickstarts. Most of them have zerombr and clearpart --all --initlabel (I think zerombr is redundant these days, but it doesn't hurt)